content/doc/administration/requirements/java.adoc

```diff
@@ -58,3 +58,18 @@ Modern versions of Jenkins controllers and Jenkins agents verify Java requiremen
 and notify users when they are launched with an unsupported version.
 
 The plugin:versioncolumn[Versions Node Monitors plugin] provides detailed Java version monitoring.
+
+== Java Development Kits Used
+
+Jenkins uses Eclipse Temurin as its primary JDK for building and testing Java based applications.
+This includes:
+
+* Container Images
```

Comment:
I think it would be worth explaining what "controller builds" are in this context.